modules/graphics/weston.ini.nix

```diff
@@ -45,8 +45,8 @@
     }
 
     {
-      path = "${gala-app}/bin/gala --enable-features=UseOzonePlatform --ozone-platform=wayland";
-      icon = "${gala-app}/gala/resources/icon-24x24.png";
+      path = "${pkgs.zathura}/bin/zathura";
```

Comment:
This change will override gala-app with zathura-app. Is that really what we want?

Reply:
Looks good now

Reply:
Fixed